pages/common/javadoc.md

```diff
@@ -0,0 +1,16 @@
+# javadoc
+
+> Generate Java API documentation in HTML format from source code.
+> More information: <https://docs.oracle.com/javase/9/javadoc/javadoc-command.htm>.
+
+- Generate documentation for Java source code and save the result in the directory:
+
+`javadoc -d {{path/to/directory/}} {{path/to/java_source_code}}`
+
+- Generate documentation with specific encoding:
+
+`javadoc -docencoding {{encoding_name}} {{java_source_code}}`
```

Comment:
Let's use UTF-8 instead of encoding_name

Reply:
@einverne - I see that you resolved this, but I don't see the change yet. Can you check ?

Reply:
Also, same here with `java_source_code` -> `path/to/java_source_code`, and also the example below this one.

Reply:
Ah, I forget to push..  :(